packages/project-access/src/project/cap.ts

```diff
@@ -156,6 +175,8 @@ export async function getCapModelAndServices(
     _logger?.info(`@sap-ux/project-access:getCapModelAndServices - Using 'cds.root': ${cds.root}`);
 
     let services = cds.compile.to.serviceinfo(model, { root: _projectRoot }) ?? [];
+    // Validate cds cache for model paths - there can be case when one of folder does not have annotations yet(for example app folder)
+    validateCdsCache(cds, modelPaths);
```

Comment:
I do not think it is a good idea to start looking into specific cache values. It's structure or logic how it is interpreted can change at any time and we'll end up having the same or worse issue. 

Clearing the cache directly is somewhat in the grey area already and we shouldn't go further. than that.